src/components/layers-panel/layers-panel.tsx

```diff
@@ -197,11 +197,33 @@ export const LayersPanel = ({
 
   useClickOutside([warningNode], () => setShowExistedError(false));
 
+  const convertUrlToRestFormat = (url: string): string => {
```

Comment:
Move to `src\utils\url-utils.ts`. Add a test

Reply:
Done.